### PR TITLE
Add OSMO, WBTC, and USDC to Dyson Protocol assetlist

### PR DESCRIPTION
**Pull request by sybilsingleton**
Hello, this adds new IBC denom metadata to Dyson.

### CODE REVIEW

File: dyson/assetlist.json [hunk 1]
```diff
@@ -19,6 +19,133 @@
...
+      "base": "ibc/9BF31153F7E37F687C75A2F310E7A4DB6E0512B065F709B5EB69FE828133AFAC",
+      "name": "Wrapped Bitcoin",
+      "display": "wbtc",
+      "symbol": "WBTC",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "osmosis",
+            "base_denom": "wbtc-satoshi",
```
Comment (JeremyParish69) [inline dyson/assetlist.json:81]: this trace is not quite right. The base denom would look more like some `ibc/<HASH>`

File: dyson/assetlist.json [hunk 2]
```diff
@@ -19,6 +19,133 @@
...
+            "channel_id": "channel-2"
+          },
+          "chain": {
+            "channel_id": "channel-208",
+            "path": "transfer/channel-2/transfer/channel-208"
```
Comment (JeremyParish69) [inline dyson/assetlist.json:86]: missing the denom at the end, after channel-208

File: dyson/assetlist.json [hunk 3]
```diff
@@ -19,6 +19,133 @@
...
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "osmosis",
+            "base_denom": "wbtc-satoshi",
+            "channel_id": "channel-2"
+          },
+          "chain": {
+            "channel_id": "channel-208",
```
Comment (JeremyParish69) [inline dyson/assetlist.json:85]: this channel needs to be updated. 208 is for Osmosis -> Axelar, not Dyson -> Osmosis

**Comment by sybilsingleton** [comment]
Ok how is this? I double checked all the IBC denoms, and added the final denom to the traces.

**Pull request merged**